content/blog/2017/2017-02-07-gsoc2017-announcement.adoc

```diff
@@ -0,0 +1,80 @@
+---
+:layout: post
+:title: "Google Summer Of Code 2017: Call for mentors"
+:tags:
+- gsoc
+- events
+- general
+- gsoc2017
+:author: oleg-nenashev
+---
+
+[NOTE]
```

Comment:
This is project news, therefore not a "guest" blog post so much. I will remove this and publish,